src/frontends/pytorch/src/transforms/prim_list_unpack_replacer.cpp

```diff
@@ -173,6 +173,32 @@ PrimListUnpackReplacer::PrimListUnpackReplacer() {
             }
         }
 
+        if (auto broadcast_tensors = cast_fw_node(input_node, "aten::broadcast_tensors")) {
+            auto tensors = cast_fw_node(broadcast_tensors->input_value(0).get_node_shared_ptr(), "prim::ListConstruct");
+            if (!tensors) {
+                add_exception_to_fw_node(input_node,
+                                         "aten::broadcast_tensors: only prim::ListConstruct supported as input.");
+                return false;
+            }
+            auto zero = opset10::Constant::create(element::i32, Shape{}, {0});
+            Output<Node> final_shape_t = zero;
+            for (auto input : tensors->inputs()) {
+                auto tensor_shape = std::make_shared<opset10::ShapeOf>(input.get_source_output());
```

Comment:
Please use rg.make instead of make_shared and call copy_runtime_info_and_name afterwards

Reply:
done